wgpu-native/src/lib.rs

```diff
@@ -85,16 +85,21 @@ pub type SamplerId = Id;
 pub type BindGroupLayoutId = Id;
 pub(crate) type BindGroupLayoutHandle = BindGroupLayout<B>;
 pub type PipelineLayoutId = Id;
+pub(crate) type PipelineLayoutHandle = PipelineLayout<B>;
 
 // Pipeline
 pub type BlendStateId = Id;
+pub(crate) type BlendStateHandle = BlendState;
 pub type DepthStencilStateId = Id;
+pub(crate) type DepthStencilStateHandle = DepthStencilState;
 pub type InputStateId = Id;
 pub type ShaderModuleId = Id;
 pub(crate) type ShaderModuleHandle = ShaderModule<B>;
 pub type AttachmentStateId = Id;
+pub(crate) type AttachmentStateHandle = AttachmentState;
 pub type ComputePipelineId = Id;
 pub type RenderPipelineId = Id;
+pub(crate) type RenderPipelineHandle = RenderPipeline<B>;
```

Comment:
`pub(crate)` doesn't give us anything for the root `lib.rs`, since all the modules see it anyway.